tf_encrypted/keras/layers/activation_layer.py

```diff
@@ -0,0 +1,29 @@
+"""Activation Layer implementation."""
+from tf_encrypted.keras.engine import Layer
+from tf_encrypted.keras import activations
+
+class Activation(Layer):
+  """Applies an activation function to an output.
+  Arguments:
+      activation: name of activation function to use or
+          TF Encrypted operation.
+  Input shape:
+      Arbitrary. Use the keyword argument `input_shape`
+      (tuple of integers, does not include the samples axis)
+      when using this layer as the first layer in a model.
+  Output shape:
+      Same shape as input.
+  """
+
+  def __init__(self, activation, **kwargs):
+    super(Activation, self).__init__(**kwargs)
+    self.activation = activations.get(activation)
+
+  def build(self, input_shape):
+    pass
+
+  def call(self, inputs):
+    return self.activation(inputs)
+
+  def compute_output_shape(self, input_shape):
+    return input_shape
```

Comment:
ideally this should be forwarded to the activation function in case it changes the shape. however, this adds some complexity so maybe it's best simply to add a comment in the docstring along the lines of "These are assumed to preserve shape."

Reply:
Do you have any in mind?  I don't know of any activation functions that change the shape of the inputs.

Reply:
oh no, simply saying that there'll be a bug if that ever happens